### PR TITLE
Task/11584

### PR DESCRIPTION
**Pull request by hoangbaovu**
### Description
Create Register Page
### Checklist
- [x] Create Register Page
- [x] Register with firebase
---
### https://edu-redmine.sun-asterisk.vn/issues/11584

### CODE REVIEW

**Comment by hoangbaovu** [comment]
ready

File: src/app/modules/register/FormRegister.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    if (password !== rePassword) {
+      return false;
+    };
+    firebaseApp.auth()
+      .createUserWithEmailAndPassword(email, password)
+      .then(data => {
+        /**
+         * TODO: Create Notify Register success
+         */
+        console.log(data.user.uid);
```
Comment (phamvanbk11) [inline src/app/modules/register/FormRegister.js]: Bỏ console

File: src/app/modules/register/FormRegister.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        usersRef.child(data.user.uid).set({
+          uid: data.user.uid,
+          firstName,
+          lastName,
+          newsletter,
+          isAdmin: false
+        })
+        console.log(data);
```
Comment (phamvanbk11) [inline src/app/modules/register/FormRegister.js]: Xóa console

File: src/app/modules/register/FormRegister.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+      })
+      .catch(e => {
+        /**
+         * TODO: Create Catch Error UI after done #task/12144
+         */
+        console.log(e);
```
Comment (phamvanbk11) [inline src/app/modules/register/FormRegister.js]: Xử lý Error luôn em

**Comment by phamvanbk11** [comment]
commented

File: src/app/modules/register/FormRegister.js (hunk 3 above)
Comment (hoangbaovu) [reply src/app/modules/register/FormRegister.js]: Em có task làm notify nên em xử lý sau a

**Comment by hoangbaovu** [comment]
ready

File: src/app/modules/register/FormRegister.js [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+          uid: data.user.uid,
+          firstName,
+          lastName,
+          newsletter,
+          isAdmin: false
+        })
+      })
+      .catch(e => {
+        /**
+         * TODO: Create Catch Error UI after done #task/12144
```
Comment (phamvanbk11) [inline src/app/modules/register/FormRegister.js]: Xử lý  Error luôn nhé

**Comment by phamvanbk11** [comment]
commented

**Comment by hoangbaovu** [comment]
ready

**Pull request merged**